scripts/eq_translations_check.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import requests
+import sys
+
+import eq_translations
+
+from requests.exceptions import ConnectionError
+
+logger = logging.getLogger(__name__)
+version = f"v{eq_translations.__version__}"
+
+try:
+    response = requests.get(
+        "https://api.github.com/repos/ONSdigital/eq-translations/releases"
+    )
+    if response.status_code == 200:
+        latest_tag = response.json()[0]["tag_name"]
+        if latest_tag != version:
```

Comment:
Our pipfile still installs from master. If we are checking the release version, then we should be installing the release version in the pipfile, not the latest commit of master?

Reply:
I also think it would be useful to log what the latest version is?.

Reply:
Ok, changed to latest release tag, logging newest version now. Thanks!👍